tutor/src/components/dropped-question.tsx

```diff
@@ -108,9 +108,39 @@ const DroppedQuestionHeadingIndicator: React.FC<DroppedQuestionHeadingIndicatorP
 })
 DroppedQuestionHeadingIndicator.displayName = 'DroppedQuestionHeadingIndicator'
 
+
+interface DroppedReviewExerciseIndicatorProps {
+    info: {
+        droppedQuestion?: {
+            drop_method?: any
+        }
+        responses: any[]
+    }
+}
+export const DroppedReviewExerciseIndicator: React.FC<DroppedReviewExerciseIndicatorProps> = observer(({
+    info,
+}) => {
+    const model = info.droppedQuestion
+
+    if (!model || !model.drop_method) return null
+
+    let students = pluralize('student', info.responses.length, true)
```

Comment:
Same thing here, the change from "all" to a number will also affect homework core which we do not want.